src/open_inwoner/openzaak/api_models.py

```diff
@@ -69,17 +71,14 @@ def process_data(self) -> dict:
         """
         Prepare data for template
         """
-        from open_inwoner.openzaak.models import StatusTranslation
-
-        status_translate = StatusTranslation.objects.get_lookup()
 
         return {
             "identification": self.identification,
             "uuid": str(self.uuid),
             "start_date": self.startdatum,
             "end_date": getattr(self, "einddatum", None),
             "description": self.zaaktype.omschrijving,
-            "current_status": status_translate.from_glom_multiple(
+            "current_status": glom_multiple(
                 self,
                 ("status.statustype.statustekst", "status.statustype.omschrijving"),
                 default="",
```

Comment:
For the `label` in `get_statuses_data` we have `default=_("No data available")`. Should we use this too for the `current_status`, or should the default be an empty string?

Reply:
Nah, it should be the same so I added it.